debs/SPECS/wazuh-agent/debian/changelog

```diff
@@ -21,6 +21,12 @@ wazuh-agent (4.4.0-RELEASE) stable; urgency=low
   * More info: https://documentation.wazuh.com/current/release-notes/
 
  -- Wazuh, Inc <info@wazuh.com>  Wed, 18 Jan 2023 12:31:50 +0000
+ 
+wazuh-agent (4.3.11-RELEASE) stable; urgency=low
+
+  * More info: https://documentation.wazuh.com/current/release-notes/
+
+ -- Wazuh, Inc <info@wazuh.com>  Thu, 08 Dec 2022 15:00:00 +0000
```

Comment:
This version was released after `4.4.1` version not `4.3.10`, see `4.2.7` and `4.3.1`, dates are not correct